### PR TITLE
Update flag prefix in prometheus docs

### LINKED ISSUE #1642: Monitor CockroachDB with Prometheus Doc Update
Re: [Monitor CockroachDB with Prometheus](https://cockroachlabs.com/docs//monitor-cockroachdb-with-prometheus.html)

## Issue Description
Prometheus changed the flag format change to -- instead of -

## Suggested Resolution 
Update examples to reflect the new flag format

### PR DESCRIPTION
**Pull request by jseldess**
Fixes #1642

### CODE REVIEW

File: v1.0/monitor-cockroachdb-with-prometheus.md [hunk 1]
```diff
@@ -36,19 +36,20 @@ Make sure you have already started a CockroachDB cluster, either [locally](start
 ## Step 2. Configure Prometheus
 
 1. Download the starter [Prometheus configuration file](https://github.com/cockroachdb/cockroach/blob/master/monitoring/prometheus.yml) and [aggregation rules](https://github.com/cockroachdb/cockroach/blob/master/monitoring/rules/aggregation.rules) for CockroachDB:
-   
+
     ~~~ shell
     # Configuration file:
     $ wget https://raw.githubusercontent.com/cockroachdb/cockroach/master/monitoring/prometheus.yml \
-    -O prometheus.y    
+    -O prometheus.y
```
Comment (mberhault) [inline v1.0/monitor-cockroachdb-with-prometheus.md]: Seems I never noticed this before. This should be `-O prometheus.yml`, that's what we use later on.

File: v1.1/monitor-cockroachdb-with-prometheus.md [hunk 2]
```diff
@@ -36,19 +36,20 @@ Make sure you have already started a CockroachDB cluster, either [locally](start
 ## Step 2. Configure Prometheus
 
 1. Download the starter [Prometheus configuration file](https://github.com/cockroachdb/cockroach/blob/master/monitoring/prometheus.yml) and [aggregation rules](https://github.com/cockroachdb/cockroach/blob/master/monitoring/rules/aggregation.rules) for CockroachDB:
-   
+
     ~~~ shell
     # Configuration file:
     $ wget https://raw.githubusercontent.com/cockroachdb/cockroach/master/monitoring/prometheus.yml \
-    -O prometheus.y    
+    -O prometheus.y
```
Comment (mberhault) [inline v1.1/monitor-cockroachdb-with-prometheus.md]: ditto

File: v1.0/monitor-cockroachdb-with-prometheus.md (hunk 1 above)
Comment (jseldess) [inline v1.0/monitor-cockroachdb-with-prometheus.md]: Done. Thanks for catching this.

File: v1.1/monitor-cockroachdb-with-prometheus.md (hunk 2 above)
Comment (jseldess) [inline v1.1/monitor-cockroachdb-with-prometheus.md]: Done.

**Pull request merged**